File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.py
def solution(sides):
    answer = 0
    i = 1
    a = max(sides)
    b = min(sides)
    s = a+b
    c = a-b
    while i < s:
        if i>c and i <s:
            answer+=1
        i +=1
    return answer

File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.java
class Solution {
    public int solution(int[] sides) {
        int answer = 0;
        int m = Math.min(sides[0], sides[1]);
        int M = Math.max(sides[0], sides[1]);
        int sh = M - m;
        int lo = M + m;
        for (int i = sh + 1; i < lo; i++) {answer++;}
        return answer;
    }
}